diff --git a/node_modules/@embedpdf/plugin-scroll/dist/index.js b/node_modules/@embedpdf/plugin-scroll/dist/index.js
index 25efbb7..eaee19a 100644
--- a/node_modules/@embedpdf/plugin-scroll/dist/index.js
+++ b/node_modules/@embedpdf/plugin-scroll/dist/index.js
@@ -17,8 +17,7 @@ class BaseScrollStrategy {
    * since items stay in a simple row without centering.
    */
   getCenteringOffsetX(_item, totalContentSize) { 
-    if (!totalContentSize || _item.width >= totalContentSize.width) return 0;
-    return (totalContentSize.width - _item.width) / 2;
+    return 0; 
   } 
   /**
    * Vertical centering offset for items shorter than the content max height.
diff --git a/node_modules/@embedpdf/plugin-scroll/dist/react/index.js b/node_modules/@embedpdf/plugin-scroll/dist/react/index.js
index ab3adf4..8569d46 100644
--- a/node_modules/@embedpdf/plugin-scroll/dist/react/index.js
+++ b/node_modules/@embedpdf/plugin-scroll/dist/react/index.js
@@ -103,7 +103,7 @@ function Scroller({ documentId, renderPage, ...props }) {
               {
                 style: { 
                   display: "flex", 
-                  justifyContent: "center",
+                  justifyContent: scrollerLayout.strategy === ScrollStrategy.Horizontal ? "center" : "flex-start", 
                   gap: scrollerLayout.pageGap 
                 }, 
                 children: item.pageLayouts.map((layout) => /* @__PURE__ */ jsx(
