diff --git a/node_modules/@embedpdf/plugin-rotate/dist/react/index.js b/node_modules/@embedpdf/plugin-rotate/dist/react/index.js
index 607afc9..831242b 100644
--- a/node_modules/@embedpdf/plugin-rotate/dist/react/index.js
+++ b/node_modules/@embedpdf/plugin-rotate/dist/react/index.js
@@ -40,6 +40,8 @@ function Rotate({
   const docRotation = (documentState == null ? void 0 : documentState.rotation) ?? 0;
   const rotation = rotationOverride !== void 0 ? rotationOverride : (pageRotation + docRotation) % 4;
   const scale = scaleOverride ?? (documentState == null ? void 0 : documentState.scale) ?? 1;
+  const containerWidth = (rotation === 1 || rotation === 3 ? height : width) * scale;
+  const containerHeight = (rotation === 1 || rotation === 3 ? width : height) * scale;
   const matrix = (rotatePlugin == null ? void 0 : rotatePlugin.getMatrixAsString({
     width: width * scale,
     height: height * scale,
@@ -52,6 +54,11 @@ function Rotate({
       ...props,
       style: {
         position: "absolute",
+        top: 0,
+        left: 0,
+        width: containerWidth,
+        height: containerHeight,
+        contain: "layout paint size",
         transformOrigin: "0 0",
         transform: matrix,
         ...style
